### DISCUSSION TITLE
OpenSHMEM F2F 2024

### DISCUSSION THREAD

**Discussion by manjugv** [openshmem-org/specification#502] [Polls]
As discussed in today's meeting, please choose the date that works best for the upcoming F2F meeting in the Maryland area. Depending on the agenda, we may be able to reduce it to two days.

**Comment by yfguo** [comment]
I choose the first option, but the second one is also OK with me,

**Reply by yfguo** [reply]
Feb 27 - Feb 29 now works better for me because a potential conflict on Jan 30 - Feb 1.

**Comment by stewartl318** [comment]
Chose second, but either is fine
